force-app/main/default/classes/BotController.cls

```diff
@@ -39,6 +44,10 @@ public with sharing class BotController {
             }
         
             for (HandlerMapping mapping : BotController.handlerMappings) {
+                this.checkAccounts();
+                
+                List<Account> accounts = [SELECT Id FROM Account];
```

Comment:
This data query is inside a loop. Please move the query outside the loop and put records....